gem2rpm/smart_proxy_plugin.spec.erb

```diff
@@ -112,8 +112,14 @@ mv %{buildroot}%{gem_instdir}/bundler.d/%{plugin_name}.rb \
 
 # sample config
 mkdir -p %{buildroot}%{foreman_proxy_settingsd_dir}
-mv %{buildroot}%{gem_instdir}/settings.d/%{plugin_name}.yml.example \
-   %{buildroot}%{foreman_proxy_settingsd_dir}/%{plugin_name}.yml
+<% config_file_paths = spec.files.select { |f| f.start_with?('config/') } +
+                       spec.files.select { |f| f.start_with?('settings.d/') } %>
+<% config_file_paths.map! { |config_file| config_file.chomp('.example') } %>
```

Comment:
This is now checking all `config/` and `settings.d/` files, chomping `.example` then putting it in `settingsd_dir` with the original name. Notice `split('/').last` works even if there were subdirectories like `settings.d/dns/route53.yml`